cmd/zvol_wait/zvol_wait

```diff
@@ -9,29 +9,25 @@ count_zvols() {
 }
 
 filter_out_zvols_with_links() {
-	while read -r zvol; do
+	echo "$zvols" | while read -r zvol; do
 		if [ ! -L "/dev/zvol/$zvol" ]; then
```

Comment:
I've tested this and it still doesn't work with zvols that have spaces. This is because the 60-zvol udev rule replaces spaces by the `+` character, so if we have a zvol named "zvol 1", it will create link `/dev/zvol/zvol+1`. This is due to udev not accepting spaces (see 5f91bd3dea49a529e87e0aa39595f074fd09736a).

We should probably do the same replacement here when checking for the /dev link.

Reply:
uhoh, good catch! updated to check for `-L "/dev/zvol/$(echo "$zvol" | tr ' ' '+' )"` instead